exe/kubernetes-deploy-restart

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
```

Comment:
Because it has completely different arguments and semantics, and it's more like a helper that we want to provide out of box.